docs/sts/kerberos.md

```diff
@@ -0,0 +1,36 @@
+# Kerberizing Minio
+
+Minio can provide temporary access credentials for users authenticated
```

Comment:
No need to tab and break the text here, you can write continuously on the same line and let it wrap on its own.